<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="apiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose where you want to collect the data:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="smearCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SMEAR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statfiCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>STATFI</string>
    </property>
   </widget>
   <widget class="QLabel" name="smearStartLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SMEAR start date: </string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GREENHOUSE GAS DATA</string>
    </property>
   </widget>
   <widget class="QLabel" name="smearEndLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SMEAR end date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statfiStartLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STATFI start year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose monitoring station:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hyytialaCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Hyytiälä</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kumpulaCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Kumpula</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="varrioCheckBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Värriö</string>
    </property>
   </widget>
   <widget class="QLabel" name="datasetLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose datasets:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inTonnesCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2 (in tonnes)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="indexedCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2 indexed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="intensityCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2 intensity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="intensityIndexedCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2 intensity indexed</string>
    </property>
   </widget>
   <widget class="QLabel" name="gasLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Greenhouse gas you want to see:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CO2CheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SO2CheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SO2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="NOxCheckBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>NOx</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose how to view the data:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rawRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>raw</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minimumRadioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>minimum</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="averageRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>average</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="maximumRadioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>maximum</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveSelectionsCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>save selections</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizePushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VISUALIZE</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="smearEndCalendar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::SingleLetterDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="smearStartCalendar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::SingleLetterDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statfiEndLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STATFI end year:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitPushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartPushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clearAllCheckBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>clear all</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="statfiStartYear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="statfiEndYear">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
